tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -1515,6 +1651,23 @@ function & lambda()
             ],
             [
                 '<?php
+
+function & lambda() {
+    return function ()
+    {
+    };
+}',
+                '<?php
+
+function & lambda()
+{
+    return function () {
+    };
+}',
+                self::$configurationOopPositionSameLine + self::$configurationCtrlStructPositionNextLine,
```

Comment:
This test is probably a good visualisation of what exactly this new option applies to – it covers everything that isn't covered by `position_after_functions_and_oop_constructs`. So this test takes something that would be unchanged by default `@PSR2` and applying the non-default configs of both this new option and `position_after_functions_and_oop_constructs` acts to invert the brace behaviour:
Because `position_after_functions_and_oop_constructs` specifically selects (non-anonymous) classes and functions, and `position_after_control_structures` will very specifically select the anonymous ones (as well as other control structures)